lib/matplotlib/pyplot.py

```diff
@@ -2686,17 +2686,28 @@ def polar(*args, **kwargs) -> list[Line2D]:
 
     call signature::
 
-      polar(theta, r, **kwargs)
-
-    Multiple *theta*, *r* arguments are supported, with format strings, as in
-    `plot`.
+      polar(theta, r, [fmt], **kwargs)
+
+    This is a convenience wrapper around `.pyplot.plot`. It ensures that the
+    current Axes is polar (or creates one if needed) and then passes all parameters
+    to ``.pyplot.plot``.
+
+    .. note::
+        When making polar plots using the :ref:`pyplot API <pyplot_interface>`,
+        ``polar()`` should typically be the first command because that makes sure
+        a polar Axes is created. Using other commands such as ``plt.title()``
+        before can lead to the implicit creation of a rectangular Axes, in which
```

Comment:
```suggestion
        before this can lead to the implicit creation of a rectangular Axes, in which
```
Tiny nit - this scans better for me.